langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -99,6 +116,16 @@ public isolated function sum(int... ns) returns int = @java:Method {
 
 # Returns the maximum of one or more int values.
 #
+# ```ballerina
+# int:max(50, 20, 30, 70, 65) ⇒ 70
+# 
+# [int, int, int] scores = [52, 95, 76];
+# int:max(int:max(...scores)) ⇒ 95
```

Comment:
```suggestion
# int:max(...scores) ⇒ 95
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/38917/commits/015f68edf3c173d04f494b82ab8c01e964211d74